2021-11-20 13:38:49,062 INFO    MainThread:21800 [wandb_setup.py:_flush():71] setting env: {}
2021-11-20 13:38:49,063 INFO    MainThread:21800 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-20 13:38:49,063 INFO    MainThread:21800 [wandb_init.py:_log_setup():357] Logging user logs to c:\vizdoom\vizdoom115pre\VDoomLearning-1\src\wandb\run-20211120_133849-37h0z0xd\logs\debug.log
2021-11-20 13:38:49,063 INFO    MainThread:21800 [wandb_init.py:_log_setup():358] Logging internal logs to c:\vizdoom\vizdoom115pre\VDoomLearning-1\src\wandb\run-20211120_133849-37h0z0xd\logs\debug-internal.log
2021-11-20 13:38:49,064 INFO    MainThread:21800 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000027860D34320>
2021-11-20 13:38:49,067 INFO    MainThread:21800 [wandb_init.py:init():390] calling init triggers
2021-11-20 13:38:49,067 INFO    MainThread:21800 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-20 13:38:49,068 INFO    MainThread:21800 [wandb_init.py:init():435] starting backend
2021-11-20 13:38:49,068 INFO    MainThread:21800 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2021-11-20 13:38:49,088 INFO    MainThread:21800 [backend.py:ensure_launched():199] starting backend process...
2021-11-20 13:38:49,301 INFO    MainThread:21800 [backend.py:ensure_launched():205] started backend process with pid: 25408
2021-11-20 13:38:49,313 INFO    MainThread:21800 [wandb_init.py:init():444] backend started and connected
2021-11-20 13:38:49,553 INFO    MainThread:21800 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-20 13:38:49,557 INFO    MainThread:21800 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-20 13:38:49,558 INFO    MainThread:21800 [wandb_init.py:init():503] updated telemetry
2021-11-20 13:38:49,628 INFO    MainThread:21800 [wandb_init.py:init():533] communicating current version
2021-11-20 13:38:54,645 INFO    MainThread:21800 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-20 13:38:58,687 INFO    MainThread:21800 [wandb_init.py:init():576] starting run threads in backend
2021-11-20 13:38:59,105 INFO    MainThread:21800 [wandb_run.py:_console_start():1693] atexit reg
2021-11-20 13:38:59,106 INFO    MainThread:21800 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-20 13:38:59,110 INFO    MainThread:21800 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-20 13:38:59,130 INFO    MainThread:21800 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-20 13:38:59,130 INFO    MainThread:21800 [wandb_init.py:init():603] run started, returning control to user process
2021-11-20 13:38:59,131 INFO    MainThread:21800 [wandb_run.py:_config_callback():911] config_cb None None {}
2021-11-20 13:38:59,383 INFO    MainThread:21800 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-20 13:38:59,383 INFO    MainThread:21800 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 13:39:56,677 INFO    MainThread:21800 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-20 13:39:56,694 INFO    MainThread:21800 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-20 13:39:56,694 INFO    MainThread:21800 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 13:40:08,639 INFO    MainThread:21800 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-20 13:40:08,640 INFO    MainThread:21800 [wandb_run.py:finish():1347] finishing run teccr/VDoomLearning/37h0z0xd
2021-11-20 13:40:08,843 INFO    MainThread:21800 [jupyter.py:save_history():429] not saving jupyter history
2021-11-20 13:40:08,843 INFO    MainThread:21800 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 13:40:08,844 INFO    MainThread:21800 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-11-20 13:40:08,844 INFO    MainThread:21800 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-20 13:40:08,846 INFO    MainThread:21800 [wandb_run.py:_restore():1635] restore
2021-11-20 13:40:09,439 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 864
}

2021-11-20 13:40:09,985 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 864
}

2021-11-20 13:40:10,553 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 4800
}

2021-11-20 13:40:10,669 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 4800
}

2021-11-20 13:40:10,780 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 4800
}

2021-11-20 13:40:10,889 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 4800
}

2021-11-20 13:40:11,006 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 4800
}

2021-11-20 13:40:11,122 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 4800
}

2021-11-20 13:40:11,226 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 864
  total_bytes: 4800
}

2021-11-20 13:40:11,336 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3610
  total_bytes: 4800
}

2021-11-20 13:40:11,452 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4063
  total_bytes: 4800
}

2021-11-20 13:40:11,574 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4063
  total_bytes: 4800
}

2021-11-20 13:40:11,681 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4800
  total_bytes: 4800
}

2021-11-20 13:40:11,790 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4800
  total_bytes: 4800
}

2021-11-20 13:40:11,907 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4800
  total_bytes: 4800
}

2021-11-20 13:40:12,016 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4800
  total_bytes: 4800
}

2021-11-20 13:40:12,124 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4800
  total_bytes: 4800
}

2021-11-20 13:40:13,574 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4800
  total_bytes: 4800
}

2021-11-20 13:40:14,540 INFO    MainThread:21800 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4800
  total_bytes: 4800
}
local_info {
}

2021-11-20 13:40:15,756 INFO    MainThread:21800 [wandb_run.py:_append_files():2061] logging synced files
